src/Command/AddMassMediaCommand.php

```diff
@@ -22,11 +25,23 @@
  */
 class AddMassMediaCommand extends BaseCommand
 {
+    /**
+     * @var EntityManagerInterface|null
+     */
+    protected $entityManager;
+
     /**
      * @var string[]
      */
     protected $setters;
 
+    public function __construct(ManagerInterface $mediaManager, Pool $pool, ?EntityManagerInterface $entityManager = null)
```

Comment:
Is this command only intended to work with the ORM manager?

Reply:
https://github.com/sonata-project/SonataMediaBundle/blob/882ad84638c7735c9d6d96f33fc18012b76bc895/src/Command/AddMassMediaCommand.php#L105-L110

Reply:
Sorry, I don't understand what you're pointing here. Other managers than ORM haven't a unit of work?